conu/apidefs/container.py

```diff
@@ -253,3 +253,11 @@ def get_file(self, file_path):
         """
         raise NotImplementedError("get_file method is not implemented")
 
+    def check_file_exists(self, filename):
```

Comment:
name `check` implies that an exception should be raised when the file does not exist (see `subprocess.call` vs `subprocess.check_call`)